MAP/src/components/TabMenu/TabMenu.js

```diff
@@ -21,11 +21,11 @@ const renderMenu = (
     t: s => {
       switch (s) {
         case 'menu.userPreferenceSection.theme.action':
-          return 'Toggle ';
+          return 'Switch to ';
         case 'menu.userPreferenceSection.theme.light':
-          return 'Light mode';
-        case 'menu.userPreferenceSection.theme.dark':
           return 'Dark mode';
+        case 'menu.userPreferenceSection.theme.dark':
+          return 'Light mode';
         default:
           return s;
       }
```

Comment:
is this block obsolete? can you try removing it to confirm it all still works?

Reply:
`locale` itself needs to be passed to `<Settings>`, but it still works without the function.